experiment/dash/dash.go

```diff
@@ -3,42 +3,132 @@ package dash
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"time"
 
+	"github.com/montanaflynn/stats"
+	"github.com/neubot/dash/client"
+	neubotModel "github.com/neubot/dash/model"
 	"github.com/ooni/probe-engine/experiment"
 	"github.com/ooni/probe-engine/experiment/handler"
-	"github.com/ooni/probe-engine/experiment/mkevent"
-	"github.com/ooni/probe-engine/measurementkit"
 	"github.com/ooni/probe-engine/model"
 	"github.com/ooni/probe-engine/session"
 )
 
 const (
-	testName    = "dash"
-	testVersion = "0.7.0"
+	testName       = "dash"
+	testVersion    = "0.8.0"
+	defaultTimeout = 55 * time.Second
+	maxRuntime     = 15.0
```

Comment:
Dash runs for 30 seconds in many cases. I'd say 30 here.

Reply:
This is the number of requests (`ch`). I think it's always 15 times.

Reply:
I don't remember where I imitated `maxRuntime`.  :))